dimod/core/sampler.py

```diff
@@ -146,6 +146,7 @@ def sample(self, bqm, **parameters):
 
             **kwargs:
                 See the implemented sampling for additional keyword definitions.
+                Unknown keywords will be swallowed with a raised warning.
```

Comment:
Much as I find this phrasing more enjoyable, I think we're stuck with something prosaic such as "Unknown keywords are accepted but a warning is raised"